opendevin/runtime/utils/runtime_build.py

```diff
@@ -6,6 +6,7 @@
 
 import docker
 import toml
+from jinja2 import Environment, FileSystemLoader
```

Comment:
Any benifit that introduce jinja2?

Reply:
It seems like it makes manipulating the Dockerfile a bit more straightforward, so this seems OK to me? Also, we already use jinja2 in OD, so it's not an extra dependency.

Reply:
yep! that's exactly the motivation - basically get rid of our existing string concatenation that makes creating dockerfile too complex and hard to maintain.